discord/guild.py

```diff
@@ -3073,25 +3073,24 @@ async def kick(self, user: Snowflake, *, reason: str | None = None) -> None:
 
     async def ban(
         self,
-        user: Snowflake,
-        *,
+        *users: Snowflake,
         delete_message_seconds: int | None = None,
         delete_message_days: Literal[0, 1, 2, 3, 4, 5, 6, 7] | None = None,
         reason: str | None = None,
     ) -> None:
```

Comment:
Update typehint

Reply:
not sure which type-hint this refers to, but also the method's return value.

Reply:
> not sure which type-hint this refers to, but also the method's return value.

That's what I was referring to